lib/STREAMS.md

```diff
@@ -132,3 +132,19 @@ Can return synchronously. Use `await` or wrap the return value in `Promise.resol
 ### readStream.peekBuffer(byteCount)
 
 Like `readStream.read` and `readStream.peek`, but returns a Buffer instead of a string.
+
+### readStream.peeker([encoding])
+### readStream.reader([encoding])
+### readStream.delimitedReader(delimiter, [encoding])
```

Comment:
Something that might be extra nice would be to call this `split` and have it return an `ObjectReadStream<string>`...